maat-court-data-api/src/main/java/gov/uk/courtdata/hardship/service/HardshipReviewService.java

```diff
@@ -0,0 +1,44 @@
+package gov.uk.courtdata.hardship.service;
+
+import gov.uk.courtdata.dto.HardshipReviewDTO;
+import gov.uk.courtdata.entity.HardshipReviewEntity;
+import gov.uk.courtdata.exception.MAATCourtDataException;
+import gov.uk.courtdata.hardship.impl.HardshipReviewImpl;
+import gov.uk.courtdata.hardship.mapper.HardshipReviewMapper;
+import gov.uk.courtdata.model.hardship.CreateHardshipReview;
+import gov.uk.courtdata.model.hardship.UpdateHardshipReview;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Service
+@RequiredArgsConstructor
+@Slf4j
+public class HardshipReviewService {
+
```

Comment:
Hi, like we discussed `rollbackFor = MAATCourtDataException.class` can be taken out. 
I would add some loggers and try/catch for exception handling in service class